### ISSUE TITLE
Yanger -t expand - does not copy over documentation-data

### ISSUE THREAD

**Issue by fnchooft** [mbj4668/yanger#46]
While transforming yang to yin with the yin-plugin ( currently not merged yet - see branch [fnchooft/](https://github.com/fnchooft/yanger/tree/add_yin_plugin_fixed) - I noticed that the expand-transformation ( -t expand ) does not copy over documentation from the groupings.

As I am generating documentation from the yin-files this is rather cumbersome.
Looking at the code I think I found a fix, but it might be good to get additional validation especially from larger yang-sets before this is merged:

The issue seems to be inside the yanger_expand module where some data is simply not copied:

```
$ git diff
diff --git a/plugins/yanger_expand.erl b/plugins/yanger_expand.erl
index 41ef195..f1dae97 100644
--- a/plugins/yanger_expand.erl
+++ b/plugins/yanger_expand.erl
@@ -288,12 +288,14 @@ sn2(#sn{kind = Kind, name = Name, children = Ch, stmt = Stmt} = SN, P, S) ->
     F11 = fun () -> mandatory(SN, S, F10) end,
     F12 = fun () -> min_elements(SN, S, F11) end,
     F13 = fun () -> max_elements(SN, S, F12) end,
+    F14 = fun () -> description(SN, S, F13) end,
+
     Arg = if Kind == 'input' orelse Kind == 'output' ->
                   [];
              true ->
                   Name
           end,
-    {yang:stmt_keyword(Stmt), Arg, pos(S), F13()}.
+    {yang:stmt_keyword(Stmt), Arg, pos(S), F14()}.
 
 update_path(SN = #sn{kind = NodeType}, P)
   when (NodeType == 'leaf') orelse
@@ -343,6 +345,10 @@ presence(SN, _State, NxtF) ->
 mandatory(SN, _State, NxtF) ->
     yang:search_all_stmts(mandatory, SN#sn.stmt) ++ NxtF().
 
+description(SN, _State, NxtF) ->
+    yang:search_all_stmts(description, SN#sn.stmt) ++ NxtF().
+
+
 min_elements(SN, _State, NxtF) ->
     yang:search_all_stmts('min-elements', SN#sn.stmt) ++ NxtF().
```

If other field should also "survive" the expansion process then maybe more needs to be done (for instance reference fields etc).

Kind regards,

Fabian

**Comment by mbj4668** [comment]
this looks fine

**Comment by mbj4668** [comment]
Fixed by #47
